metricflow/model/parsing/dir_to_model.py

```diff
@@ -171,23 +177,20 @@ def parse_config_yaml(
                 major_version = version.major
 
                 if major_version != 0:
-                    errors.append(
-                        str(
-                            ParsingException(
-                                f"Unsupported version {version} in config document.",
-                                config_filepath=config_yaml.filepath,
-                            )
+                    issues.append(
+                        ValidationError(
+                            context=FileContext(file_name=ctx.filename, line_number=ctx.start_line),
+                            message=f"Unsupported version {version} in config document.",
```

Comment:
See like this doesn't continue or skip or anything, so we'll get results offset from errors by one per document with VERSION_KEY set to the wrong thing.